Bump actions/dependency-review-action from 4.5.0 to 4.6.0 (#295)
rsksmart/devportal: tutorials/tokens/create-a-collectable-token.md

description: "How to create your own collectable token on RSK network using Truffle framework, Open Zeppelin libraries and a react frontend."
---

> In this tutorial, you will learn about blockchain programming from scratch by building a fully decentralized application step-by-step. 
> You will also learn how to create your own collectable token on the RSK blockchain network using the Truffle framework, Open Zeppelin(OZ) libraries, and build a frontend with the `create-react-app`.

Reviewer: ```suggestion
In this tutorial, you will learn about blockchain programming from scratch by building a fully decentralized application (DApp), step by step. 
You will also learn how to create your own collectable token on the RSK blockchain network using the Truffle framework, Open Zeppelin (OZ) libraries, and build a front end with React, using `create-react-app`.
```

### Colors

> In this tutorial, We are going to create a NFT to represent our collectible color tokens. 

Reviewer: ```suggestion
In this tutorial, we are going to create an NFT to represent our collectible color tokens.
```

## Understanding App.js

Import web3 here:

```javascript
import Web3 from 'web3'
```

> This part is connected to RSK testnet using the web3 wallet injected, in this case, Metamask:

Reviewer: ```suggestion
This part is connected to the RSK Testnet using the injected web3 provider, in this case, MetaMask:
```

# Congratulations!

> Hope it was easy for you to create a NFT? 

Reviewer: ```suggestion
Hope it was easy for you to create a NFT!
```

> I showed you how to connect Truffle to the RSK network and deploy your own NFT using the Open Zeppelin(OZ) libraries, and that they work on the RSK network.

Reviewer: ```suggestion
I showed you how to connect Truffle to the RSK network and deploy your own NFT using the OZ libraries, and that they work on the RSK network.
```
